nuitka/utils/Distributions.py

```diff
@@ -82,6 +82,8 @@ def getDistributionTopLevelPackageNames(distribution):
                 continue
             if first_path_element == "__pycache__":
                 continue
+            if first_path_element.endswith("."):
```

Comment:
I was going to suggest using the existing function:

```
def checkModuleName(value):
    return ".." not in str(value) and not (
        str(value).endswith(".") or str(value) == "."
    )
```

However, for reasons of using it in importing logic, it allows `"."`, but I guess that could also be done.